topic_folders/communications/tools/slack-and-email.md

```diff
@@ -5,3 +5,30 @@ There are many ways in which you can join our conversations:
 - The [Discuss email list](http://carpentries.topicbox.com/groups/discuss), which community members are welcome to join and post to.
 - Our [other email lists](https://carpentries.org/community/#mailing-lists) are a combination of regional and topic-specific lists.
 - Our [Slack channel](https://swcarpentry.slack.com/), which community members are welcome to [join](https://swc-slack-invite.herokuapp.com/).  Click on "Channels" in the left panel to view all existing channels.
+
+Below are descriptions of selected channels found on TopicBox (as discussed in this [blog post](https://carpentries.org/blog/2020/04/channels-to-join-topicbox/)):
+
+- [**Instructor Trainers**](https://carpentries.topicbox.com/groups/trainers) - A list for topics related to the community of Trainers that prepare new instructors in teaching Data, Software and Library Carpentries lessons. 
+- [**Discuss**](https://carpentries.topicbox.com/groups/discuss) - A group for general discussion of Carpentry teaching practices and community activities and concerns. 
+- [**Discussion Hosts**](https://carpentries.topicbox.com/groups/discussion-hosts) - The community of instructors that hosts community discussion sessions. 
+- [**Instructors**](https://carpentries.topicbox.com/groups/instructors) - An admin-only list where opportunities for badged instructors to teach are announced. 
```

Comment:
do we mean to say 'an invite-only' list?

Reply:
@serahrono They don't get an invite, but need to approved. I got this verbiage from the TopicBox channel, so perhaps we need to update that too?